### ISSUE TITLE
Alternate Title does not capture special characters.

### ISSUE THREAD

**Issue by cdeluca495** [ndlib/sipity#741]
Alt title does not display Alt characters as typed (as opposed to the main title and advisor names). The ü got dropped from "kücken".
Example work:
https://sipity-staging.library.nd.edu/works/dev00125r3x

**Comment by jeremyf** [comment]
When I entered diacritics the characters were preserved. See https://sipity-staging.library.nd.edu/work_submissions/dev00125s2w

**Comment by jeremyf** [comment]
Digging through the logs:

```console
I, [2015-06-12T12:14:52.903575 #29640]  INFO -- :   Parameters: {"utf8"=>"✓", "authenticity_token"=>"UL/e/S5OXgh7O+rDWDthITaNzviFzwqzoZ0sDaR8pYpgxoN7bqpcXFaflWSpw2RVVAP8yYbosOZAVLxGU0vhOA==", "work"=>{"title"=>"<p>Shari's test 6-12-15</p>", "abstract"=>"<p>My multi-paragraph abstract starts here...</p><p>And finishes here.</p>", "alternate_title"=>"<p>Das kleine Kcken piept</p>"}, "form/describe/submit"=>"Save and continue", "work_id"=>"dev00125r3x", "processing_action_name"=>"describe"}
I, [2015-06-12T12:15:38.043410 #29640]  INFO -- :   Parameters: {"utf8"=>"✓", "authenticity_token"=>"vHR25gOn2GLshPCA0wGiDaAF4ISLC6d7ssFM/mOSTNqMDStgQ0PaNsEgjyci+ad5wovStYgsHS5TCNy1lKUIaA==", "work"=>{"title"=>"<p>Shari's test 6-12-15</p>", "abstract"=>"<p>My multi-paragraph abstract starts here...</p><p>And finishes here.</p>", "alternate_title"=>"<p>Das kleine Kcken piept</p>"}, "form/describe/submit"=>"Save and continue", "work_id"=>"dev00125r3x", "processing_action_name"=>"describe"}
I, [2015-06-12T12:25:42.162891 #5946]  INFO -- :   Parameters: {"utf8"=>"✓", "authenticity_token"=>"BDIcC2QDF2wRp0JFsBCaIOuqhm3fOMsS99TjBUwZgMY0S0GNJOcVODwDPeJB6J9UiSS0XNwfcUcWHXNOuy7EdA==", "work"=>{"title"=>"<p>Shari's test 6-12-15 adding special character to test — Cesár</p>", "abstract"=>"<p>My multi-paragraph abstract starts here...</p><p>And finishes here.</p>", "alternate_title"=>"<p>Das kleine Kcken piept</p>"}, "form/describe/submit"=>"Save and continue", "work_id"=>"dev00125r3x", "processing_action_name"=>"describe"}
```

The `ü` of `Kücken` was not sent along as part of the request. It could be the rich text editor

**Comment by danhorst** [comment]
Diacritics are supported—as long as they are entered directly into the text fields.
![741-diacritic-support](https://cloud.githubusercontent.com/assets/2133/8167046/af304f04-1369-11e5-8141-f1fdb430fb37.png)
There may be some on-paste sanitization happening courtesy of [Redactor](http://imperavi.com/redactor/).

**Comment by cdeluca495** [comment]
Several attempts to reproduce have failed.  Email from Shari Sweet:
------------------
Chris,

I started a new deposit to walk through the steps (Shari's test 6-17), and the alt character IS showing up in the alternate title field today. I used the same insert method (ALT + 129) and typed in the title, and I'm using the same computer: Chrome Version 43.0.2357.124 m and Windows 7. 

I don't know what changed between today and the previous test on the 12th, but that field does not seem to be dropping the alt characters anymore. 

Thanks,

-Shari Sweet
--------------------

Issue closed.
